Makefile

```diff
@@ -29,8 +29,12 @@ clean:
 deps:
 	test -n "$(GOPATH)" # Make sure GOPATH defined
 	curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s -- -b ${GOPATH}/bin v1.12.3
+	go get -u github.com/vbatts/git-validation
```

Comment:
These lines cause my `go.mod` and `go.sum` to get modified, because running `go get` when modules are enabled is how you add dependencies to a module.  I think we don't want to do that.

Reply:
Thanks!